nix/commands/__tools.sh

```diff
@@ -2,15 +2,19 @@
 
 # constants
 
-declare -r COLOR_CMD='\033[0;33m' # yellow
-declare -r COLOR_DIR='\033[0;36m' # cyan -- same as chalk use in common/
-declare -r COLOR_ERR='\033[0;31m' # red
-declare -r COLOR_OFF='\033[0m' # no-color
-declare -r COLOR_OK='\033[0;32m' # green
-declare -r COLOR_WARN='\033[1;31m' # light red
-
-declare __basedir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
-declare -r ELECTRON_GN_SCRIPTS_ROOT="$(git -C "${__basedir}" rev-parse --show-toplevel)"
+declare -r -x COLOR_CMD='\033[0;33m' # yellow
+declare -r -x COLOR_DIR='\033[0;36m' # cyan -- same as chalk use in common/
+declare -r -x COLOR_ERR='\033[0;31m' # red
+declare -r -x COLOR_OFF='\033[0m' # no-color
+declare -r -x COLOR_OK='\033[0;32m' # green
+declare -r -x COLOR_WARN='\033[1;31m' # light red
```

Comment:
All of the COLOR_ variables are in the same bucket as `ELECTRON_EXEC` and its suggestion applies here. Probably something like

```sh
# used by scripts that source this file, so disable "unused variable" warnings
# shellcheck disable=SC2034
declare -r COLOR_CMD='\033[0;33m' # yellow
# shellcheck disable=SC2034
declare -r COLOR_DIR='\033[0;36m' # cyan -- same as chalk use in common/
...
```